volumes/vds/readme.md

```diff
@@ -0,0 +1,255 @@
+
+## vds.veh_length_15min
+Summarization of vdsvehicledata with count of observation (vehicle) lengths grouped by 15 min / 1m length / vds_id. 
+
+Row count: 4,622,437
+| column_name    | data_type                   | sample              | description   |
+|:---------------|:----------------------------|:--------------------|:--------------|
+| division_id    | smallint                    | 2                   |               |
+| vds_id         | integer                     | 3                   |               |
+| datetime_15min | timestamp without time zone | 2023-06-13 00:00:00 |               |
+| length_meter   | smallint                    | 0                   | 1m length bins, rounded down. |
+| count          | smallint                    | 3                   |               |
+| total_count    | smallint                    | 5                   | Use count::numeric/total_count to get proportion. |
+| uid            | bigint                      | 4866932             |               |
+
+
+## vds.raw_vdsdata_div8001
+A sample of 1 day of vdsdata for sensors from division_id 8001. The data is mostly blank rows and may not be of any utility. Main `raw_vdsdata` table is now filtered to only division_id = 2. 
+
+Row count: 601,460
+| column_name       | data_type                   | sample              | description   |
+|:------------------|:----------------------------|:--------------------|:--------------|
+| division_id       | smallint                    | 8001                |               |
+| vds_id            | integer                     | 3437088             |               |
+| datetime_20sec    | timestamp without time zone | 2023-06-28 00:00:02 |               |
+| datetime_15min    | timestamp without time zone | 2023-06-28 00:00:00 |               |
+| lane              | integer                     | 1                   |               |
+| speed_kmh         | double precision            |                     |               |
+| volume_veh_per_hr | integer                     | 0                   |               |
+| occupancy_percent | double precision            | 0.0                 |               |
+
+### Special Function and Preemption detectors (Division 8001): 
+These types of sensors include transit preemption, fire emergency services preemption. The one day sample we have only contains 22 detections which doesn't seem accurate. Sample query: 
+```
+SELECT DISTINCT ON (vds_id, datetime_20sec)
+    c.detector_id, d.* 
+FROM vds.raw_vdsdata_div8001 AS d
+LEFT JOIN vds.vdsconfig AS c
+    ON d.vds_id = c.vds_id
+--WHERE c.detector_id SIMILAR TO 'PX[1-9]{4}-DET%'
+WHERE (detector_id SIMILAR TO 'PX[1-9]{4}-SF%' OR detector_id SIMILAR TO 'PX[1-9]{4}-PE%')
+    AND (volume_veh_per_hr > 0 OR occupancy_percent > 0)
+ORDER BY vds_id, datetime_20sec, c.start_timestamp DESC
+LIMIT 1000
+```
+
+
+# Dag Design 
+
+## vds_pull_vdsdata [../../../dags/vds_pull_vdsdata.py]
```

Comment:
If you want to link the text to another file, I think you should have it like this: `[text](link)`, e.g. in this example: `[vds_pull_vdsdata](../../../dags/vds_pull_vdsdata.py)`. But how would this look like as a heading 🤔